src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'cd-info-card',
+  templateUrl: './info-card.component.html',
+  styleUrls: ['./info-card.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Please don't use this encapsulation.
This will result in the css from this component to be applied to the whole document.